subsys/bluetooth/controller/ll_sw/ull_df.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2018-2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SUBSYS_BLUETOOTH_CONTROLLER_LL_SW_ULL_DF_H_
+#define ZEPHYR_SUBSYS_BLUETOOTH_CONTROLLER_LL_SW_ULL_DF_H_
+
+#include <sys/util.h>
+
```

Comment:
```suggestion
```

Keep the include in the c file using this header.

Reply:
include moved to *.c file